### PR TITLE
toggl: Add version 7.5.363

### PR DESCRIPTION
**Pull request by dtger**

### CODE REVIEW

**Comment by dtger** [comment]
Not sure what changes need to happen as the error seems with the automated check on PS6? Need a little more handholding here, apologies.

File: bucket/toggl.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/toggl-open-source/toggldesktop/releases/download/v$version/TogglDesktopInstaller-x64-$version.exe"
+            }
+        }
+    }
+}
```
Comment (Ash258) [inline bucket/toggl.json:24]: For the checkver I need to find better way

```suggestion
{
    "version": "7.5.162",
    "description": "Time Tracking for Tech Companies",
    "homepage": "https://toggl.com/toggl-desktop",
    "license": {
        "identifier": "Proprietary",
        "url": "https://www.toggl.com/legal"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/toggl-open-source/toggldesktop/releases/download/v7.5.162/TogglDesktopInstaller-x64-7.5.162.exe#/dl.7z",
            "hash": "5967842808bb73a7705afdca6b5fd7553da5f4967b864e7e0f6d83525cfd9e13"
        },
        "32bit": {
            "url": "https://github.com/toggl-open-source/toggldesktop/releases/download/v7.5.162/TogglDesktopInstaller-7.5.162.exe#/dl.7z",
            "hash": "b1dcab85ebc2f96cb4f87a21f0819d053932e66894e6304bb7a1cca9c882f6d8"
        }
    },
    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstal*\" -Recurse",
    "shortcuts": [
        [
            "TogglDesktop.exe",
            "Toggl Desktop"
        ]
    ],
    "checkver": {
        "url": "https://toggl-open-source.github.io/toggldesktop/assets/changelog/win.json",
        "regex": "\"version\":\"([\\d.]+)\""
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/toggl-open-source/toggldesktop/releases/download/v$version/TogglDesktopInstaller-x64-$version.exe#/dl.7z"
            },
            "32bit": {
                "url": "https://github.com/toggl-open-source/toggldesktop/releases/download/v$version/TogglDesktopInstaller-$version.exe#/dl.7z"
            }
        }
    }
}

```

Comment (dtger) [reply bucket/toggl.json:24]: Mate, what's with the attitude? I'm new and still trying to figure things out here (github and scoop both), there's no need to react quite as harshly. 

Thank you for making suggestions above, i'll check those out.

Comment (linsui) [reply bucket/toggl.json:24]:
```json
    "checkver": {
        "url": "https://toggl-open-source.github.io/toggldesktop/download/windows64-stable/",
        "regex": "x64-([\\d.]+)\\.exe"
    },
```

This works for me.

File: bucket/toggl.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://www.toggl.com/",
+    "description": "Time Tracking for Tech Companies",
+    "version": "7.5.119",
+    "license": "Proprietary",
```
Comment (pratikpc) [inline bucket/toggl.json:5]: License is BSD-3-Clause apparently

https://github.com/toggl-open-source/toggldesktop/blob/master/LICENSE

**Comment by issaclin32** [comment]
By the way, *persist* is not needed because config is stored at `$Env:LocalAppdata\TogglDesktop`.

**Comment by rashil2000** [comment]
Isn't 7.5.454 a pre-release?

**Comment by rashil2000** [comment]
Sorry this is a little confusing. The GitHub repo mentions latest stable release as 7.5.363 (released Oct 30, 2020), but their website says 7.5.445 (released Mar 25, 2021). See - https://toggl.github.io/toggldesktop/

The strange part is that the homepage doesn't contain 7.5.363 release.

**Comment by issaclin32** [comment]
The problem is that the changelog (https://toggl.github.io/toggldesktop/) is generated by javascript.
It cannot be parsed by *checkver*.

**Comment by niheaven** [comment]
The changelog is parsed from https://toggl-open-source.github.io/toggldesktop/assets/changelog/win.json, the version number could be extracted from here.

**Pull request merged**